docs/modules/pipelines/pages/submitting-jobs.adoc

```diff
@@ -121,6 +121,40 @@ You cannot upload the following classes using the Jet API or the CLI. These clas
 * Map features such as EntryProcessor or MapLoader and MapStore
 ====
 
+=== Isolated Jobs
+
+To activate isolated jobs, your license key must include `Advanced Compute`.
+
+You can run Jet jobs on a specific set of members. Jobs run this way are known as isolated jobs.
+
```

Comment:
We should include some information on the guarantees of "isolated jobs" here - we are able to maintain the majority of normal job guarantees, but there are some caveats. @Fly-Style could you please provide more detail to add here?

Reply:
Sure.  For "Isolated Jobs" we support 
- auto-scaling
- fault-tolerance (both `AT_LEAST_ONCE` and `EXACTLY_ONCE`)
- split-brain protection
- metrics

Reply:
What is not supported for "isolated jobs" but is for normal jobs? @Fly-Style

Reply:
Updated with supported list